packages/react/src/prefabs/MediaDeviceMenu.tsx

```diff
@@ -93,12 +96,12 @@ export function MediaDeviceMenu({
 
   React.useEffect(() => {
     document.addEventListener<'click'>('click', handleClickOutside);
-    window.addEventListener<'resize'>('resize', () => setUpdateRequired(true));
+    // window.addEventListener<'resize'>('resize', () => setUpdateRequired(true));
```

Comment:
Is this intentional or should it be removed?

Reply:
the idea is that the responsibility for this is being shifted to the `autoUpdate` helper of the floating ui lib. 
this includes screen resizing and a bunch of other things, see https://floating-ui.com/docs/autoupdate